x/transferinject/ibc_middleware_test.go

```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
```

Comment:
this test file should also be split as the `ibc_middleware.go`, so we know where the the tests for the corresponding implementation files are

Reply:
good point Ill change it 👍